<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSBasicStatusPanel</class>
 <widget class="QWidget" name="PLSBasicStatusPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>232</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="0" column="10" colspan="7">
    <widget class="QLabel" name="streamHdr">
     <property name="styleSheet">
      <string notr="true">min-width: 290;</string>
     </property>
     <property name="text">
      <string>Basic.Stats.Stream.Status</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="cpuUsage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">0%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="5">
    <widget class="QLabel" name="performanceHdr">
     <property name="styleSheet">
      <string notr="true">min-width: 290;</string>
     </property>
     <property name="text">
      <string>Basic.Stats.Performance</string>
     </property>
    </widget>
   </item>
   <item row="5" column="14">
    <widget class="QLabel" name="streamBitrateLabelV">
     <property name="text">
      <string>Basic.Stats.Output.Stream</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="13">
    <spacer name="horizontalSpacer_21">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>7</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="14">
    <widget class="QLabel" name="StreamMegabytesLabel">
     <property name="text">
      <string>Basic.Stats.MegabytesStream</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="12" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="networkFramedropStateV">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QLabel" name="networkStateImage">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="renderingFramedropLabel">
     <property name="text">
      <string>Basic.Stats.RenderingFramedrop</string>
     </property>
    </widget>
   </item>
   <item row="3" column="20">
    <widget class="QLabel" name="recordedMegabytes">
     <property name="text">
      <string notr="true">0.0 MiB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="18">
    <widget class="QLabel" name="diskAvailableLabel">
     <property name="text">
      <string>Basic.Stats.HDDSpaceAvailable</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="renderingFramedropState"/>
   </item>
   <item row="4" column="4" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="encodingFramedropStateV">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="14">
    <widget class="QLabel" name="networkFramedropLabelV">
     <property name="text">
      <string>Basic.Stats.NetorkFramedrop</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="16">
    <widget class="QLabel" name="networkFramedrop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">0 / 0 (0.0%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="renderingFramedrop">
     <property name="text">
      <string notr="true">0 / 0 (0.0%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="20">
    <widget class="QLabel" name="recordingBitrate">
     <property name="text">
      <string notr="true">0 kb/s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="realtimeFramerateLabel">
     <property name="text">
      <string>Basic.Stats.realtimeFPS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="12" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="networkFramedropState">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="realtimeFramerate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">0 / 0 fps</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="resourcesHdr">
     <property name="styleSheet">
      <string notr="true">min-width: 180;</string>
     </property>
     <property name="text">
      <string>Basic.Stats.Resources</string>
     </property>
    </widget>
   </item>
   <item row="1" column="19">
    <spacer name="horizontalSpacer_17">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="12" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="streamBitrateStateV">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QLabel" name="networkStateImageV">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="12" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="streamBitrateState">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="gpuUsageLabel">
     <property name="text">
      <string>Basic.Stats.GPUUsage</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="16">
    <widget class="QLabel" name="streamMegabytesV">
     <property name="text">
      <string notr="true">0.0 MiB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="realtimeFramerateState"/>
   </item>
   <item row="4" column="6">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="encodingFramedropImageV">
        <property name="active" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_31">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>2</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="encodingFramedropLabelV">
        <property name="text">
         <string>Basic.Stats.EncodingFramedrop</string>
        </property>
        <property name="active" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="18">
    <widget class="QLabel" name="recordedMegabytesLabel">
     <property name="text">
      <string>Basic.Stats.MegabytesRecord</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QLabel" name="encodingFramedrop">
     <property name="text">
      <string notr="true">0 / 0 (0.0%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="19">
    <spacer name="horizontalSpacer_19">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="14">
    <widget class="QLabel" name="streamBitrateLabel">
     <property name="text">
      <string>Basic.Stats.Output.Stream</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="gpuUsage">
     <property name="text">
      <string notr="true">0%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="14">
    <widget class="QLabel" name="StreamMegabytesLabelV">
     <property name="text">
      <string>Basic.Stats.MegabytesStream</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="19">
    <spacer name="horizontalSpacer_18">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="16">
    <widget class="QLabel" name="streamMegabytes">
     <property name="text">
      <string notr="true">0.0 MiB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4" alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="encodingFramedropState">
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLabel" name="encodingFramedropV">
     <property name="text">
      <string notr="true">0 / 0 (0.0%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="20">
    <widget class="QLabel" name="diskAvailable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">0 GB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="16">
    <widget class="QLabel" name="streamBitrate">
     <property name="text">
      <string notr="true">0 kb/s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <spacer name="horizontalSpacer_20">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>2</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6">
    <widget class="QWidget" name="widget_1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="encodingFramedropImage">
        <property name="active" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_30">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>2</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="encodingFramedropLabel">
        <property name="text">
         <string>Basic.Stats.EncodingFramedrop</string>
        </property>
        <property name="active" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="18" colspan="3">
    <widget class="QLabel" name="recordHdr">
     <property name="styleSheet">
      <string notr="true">min-width: 210;</string>
     </property>
     <property name="text">
      <string>Basic.Stats.Record.Status</string>
     </property>
    </widget>
   </item>
   <item row="1" column="15">
    <spacer name="horizontalSpacer_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="memoryUsage">
     <property name="text">
      <string notr="true">0 MB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="17">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="memoryUsageLabel">
     <property name="text">
      <string>Basic.Stats.MemoryUsage</string>
     </property>
    </widget>
   </item>
   <item row="5" column="16">
    <widget class="QLabel" name="streamBitrateV">
     <property name="text">
      <string notr="true">0 kb/s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="16">
    <widget class="QLabel" name="networkFramedropV">
     <property name="text">
      <string notr="true">0 / 0 (0.0%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="14">
    <widget class="QLabel" name="networkFramedropLabel">
     <property name="text">
      <string>Basic.Stats.NetorkFramedrop</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cpuUsageLabel">
     <property name="text">
      <string>Basic.Stats.CPUUsage</string>
     </property>
    </widget>
   </item>
   <item row="2" column="18">
    <widget class="QLabel" name="recordingBitrateLabel">
     <property name="text">
      <string>Basic.Stats.Output.Recording</string>
     </property>
     <property name="active" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_26">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>7</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="5">
    <spacer name="horizontalSpacer_27">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>7</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="5">
    <spacer name="horizontalSpacer_28">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5">
    <spacer name="horizontalSpacer_29">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
